docs/source/user/stencil.rst

```diff
@@ -171,10 +172,10 @@ specified neighborhood, **the behavior is undefined.**
 
 .. _stencil-mode:
 
-``mode``
+``func_or_mode``
```

Comment:
Title underline needs extending to the same number of chars as the title. Docs are built with warning-as-error so I think this will fail to build.

Reply:
@AaronCritchley fixed this in b7d223f.

Reply:
Thank you for fixing, sorry for not catching this, I didn't build the docs after my change, lesson learnt 😄

Reply:
No problem. Thanks for instigating these changes, the stencil options read better now!